### PR TITLE
Remove old exports

### PR DESCRIPTION
**Pull request by kfichter**
## Description
We have a bunch of old stuff here that we don't seem to be using. I got rid of those old things, feel free to let me know if you're making use of any of these things and I don't know about it.

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: packages/core-utils/src/coders/sequencer-batch.ts [hunk 1]
```diff
@@ -124,3 +125,8 @@ export const decodeAppendSequencerBatch = (
     transactions,
   }
 }
+
+export const sequencerBatchAppendCoder: Coder = {
```
Comment (kfichter) [inline packages/core-utils/src/coders/sequencer-batch.ts:129]: I figured I would export this as a coder to be nice.

**Comment by kfichter** [comment]
>.> it's failing because of our custom provider. Is there any reason why we still need that?

**Review by karlfloersch — COMMENTED** [review]
@tynes you might want to give this a look over as well but there’s nothing in here that I can see that I’m using. Not using those simple http server libraries anymore! That’s some OG code though!!

**Comment by kfichter** [comment]
Closing in favor of simply using the new `core-utils` repo which gets rid of all of these.

**Pull request closed without merging**